libs/core/src/lib/datetime-picker/datetime-picker.component.ts

```diff
@@ -283,6 +283,11 @@ export class DatetimePickerComponent<D>
     // eslint-disable-next-line @angular-eslint/no-output-on-prefix
     readonly onClose: EventEmitter<void> = new EventEmitter<void>();
 
+    /** Event emitted when datepicker changes it's "touched" state. */
+    @Output()
+    // eslint-disable-next-line @angular-eslint/no-output-on-prefix
+    readonly touched: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
Why do we need this?

Reply:
`fpd-datetimepicker` uses core implementation under the hood. Using this output here https://github.com/SAP/fundamental-ngx/blob/33423dd585bef9922bec19c9318e83743556987f/libs/platform/src/lib/form/datetime-picker/datetime-picker.component.html#L38

There're few cases when core component is being marked as touched and I needed a way to react to all of them